### PR TITLE
fix: Gatling test initial data load

### PR DESCRIPTION
**Pull request by abhijitdiwan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/AM-362
https://tools.hmcts.net/jira/browse/AM-380

### Change description ###
fix: Gatling test initial data load

### CODE REVIEW

File: src/gatling/simulations/uk/gov/hmcts/reform/amlib/performance/scenarios/BasicScenarios.scala [hunk 1]
```diff
@@ -34,7 +34,7 @@ object BasicScenarios {
 
   val returnResourceAccessors: ScenarioBuilder = scenario("Return Resource Accessors")
     .forever(
-      feed(ReturnResourceAccessors.feed)
+      feed(CreateResourceAccess.feed)
         .exec(AccessManagement.createResourceAssess)
```
Comment (shashank-rastogi) [inline src/gatling/simulations/uk/gov/hmcts/reform/amlib/performance/scenarios/BasicScenarios.scala]: i  am not sure why this change is required?

File: Jenkinsfile_nightly [hunk 2]
```diff
@@ -59,9 +68,45 @@ withNightlyPipeline(type, product, component) {
...
+          sh "/nightlyperformancedata/load-or-delete-performance-testdata.sh \
+                        ${WORKSPACE} \
+                        'delete' \
+                        ${DATABASE_USER} \
+                        ${DATABASE_NAME} \
+                        ${DATABASE_HOST} \
+                        ${DATABASE_PASS} \
+                        ${DATABASE_PORT} \
+                        ${params.ENVIRONMENT}"
+        }
```
Comment (shashank-rastogi) [inline Jenkinsfile_nightly]: as this could be used to set up data for other types of tests like functional tests as well, we should try to create a function and pass parameter like test type and script to be executed and try to avoid writing same code multiple times

**Review by shashank-rastogi — CHANGES_REQUESTED** [review]

File: src/gatling/simulations/uk/gov/hmcts/reform/amlib/performance/scenarios/BasicScenarios.scala (hunk 1 above)
Comment (abhijitdiwan) [reply src/gatling/simulations/uk/gov/hmcts/reform/amlib/performance/scenarios/BasicScenarios.scala]: This is just scala class setting default values for feed def feed: Feeder[String] = Iterator.continually {
    Map(
      "resourceId" -> randomUUID,
      "accessorId" -> randomUUID
    )
  }

we don't have any ReturnResourceAccessors feed.  and we can use the same.

File: Jenkinsfile_nightly (hunk 2 above)
Comment (abhijitdiwan) [reply Jenkinsfile_nightly]: done

**Review by shashank-rastogi — APPROVED** [review]
